README.md

```diff
@@ -54,6 +54,13 @@ We welcome your contributions! Please see the [contribution guidelines](CONTRIBU
 
 For SDK feedback or to report a bug, please file a [GitHub Issue](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/issues). For general suggestions or ideas, visit our [feedback forum](https://aka.ms/azurekinectfeedback).
 
+## Sample Code
+
+There are several places where the sample code can be found.
+
+- In this repository: \\Azure-Kinect-Sensor-SDK\examples- each example has a read me page that describes it and the steps to set it up.
```

Comment:
- Please make \\Azure-Kinect-Sensor-SDK\examples a link to https://github.com/microsoft/Azure-Kinect-Sensor-SDK/tree/develop/examples.
- read me should be readme